tests/test_app.py

```diff
@@ -37,18 +37,11 @@
 
 decoder = json.JSONDecoder()
 
-config = configparser.ConfigParser()
-config.read(os.environ["KB_DEPLOYMENT_CONFIG"])
+config = StagingServiceConfig(os.environ["KB_DEPLOYMENT_CONFIG"])
```

Comment:
```suggestion
CONFIG = StagingServiceConfig(os.environ["KB_DEPLOYMENT_CONFIG"])
```

It's a global, so...

Reply:
:+1: